backend/apps/ecommerce/types.py

```diff
@@ -29,6 +30,12 @@ class Meta:
             "timestamp",
         ]
 
+    @staticmethod
+    def resolve_payment_status(parent: Order, info) -> str:
+        if isinstance(parent.payment_status, Enum):
+            return parent.payment_status.value
+        return parent.payment_status
+
```

Comment:
Related to a change in how choice fields are handled, however, I am unable to understand why it has to be so complicated.